resources/ext/ezbake.conf

```diff
@@ -1,7 +1,22 @@
+# NOTE: I'm no longer certain whether or not it actually makes sense to keep this
+# file in this repo.  When we have to do preinst stuff, we get into a situation
+# where we need to do some kind of variable interpolation into the commands
+# that we're expressing here... and it's starting to feel like we just have
+# too many levels of indirection going on.  Maybe this stuff should just be
+# kept directly in the ezbake repo.
 ezbake: {
    pe: {}
    foss: {
       redhat: { dependencies: ["puppet-agent"],
+               # This is terrible, but we need write access to puppet's
+               # var/conf dirs, so we need to add ourselves to the group.
+               # Then we need to chmod some dirs until the Puppet packaging
+               # is changed to allow group write; Haus said that this
+               # has probably the way to do it for now.  There might be a better
+               # way to get rid of the hard-coded paths here, but I don't
+               # know it.
+               preinst: ["install --group={{user}} --owner={{user}} -d /var/lib/puppet/jruby-gems",
```

Comment:
@jeffmccune Seems like these don't make sense to include here anymore?  I think we decided that the gems directory would be created by `puppetserver gem` later on anyway when needed so we don't need to do this in packaging.  The manifests directory location here wouldn't be right either and I thought the creation of the production environment was going to move over to the puppet-agent package now?